src/Driver/Configurations.jl

```diff
@@ -0,0 +1,131 @@
+"""
+    CLIMA driver configurations
+
+Use to run CLIMA using the CLIMA driver's `CLIMA.invoke()`. User-customized
+configurations can use these as templates.
+"""
+
+using MPI
+
+using CLIMA
+using CLIMA.DGmethods
+using CLIMA.DGmethods.NumericalFluxes
+using CLIMA.Mesh.Topologies
+using CLIMA.Mesh.Grids
+using CLIMA.MPIStateArrays
+using CLIMA.Atmos
+using CLIMA.ODESolvers
+using CLIMA.PlanetParameters
+
+mutable struct DriverConfiguration
+    # initialized at construction
+    name::String
+    FT::DataType
+    array_type
+    #
+    # AtmosModel details
+    bl::BalanceLaw
+    numfluxnondiff::NumericalFluxNonDiffusive
+    numfluxdiff::NumericalFluxDiffusive
+    gradnumflux::GradNumericalPenalty
+
+    # initialized after construction
+    #
+    # execution details
+    mpicomm::Union{Missing,MPI.Comm}
```

Comment:
Just curious: is `Missing` preferred over `Nothing` here? And if so, why?

Reply:
Could use either in this case. The explanation of the difference [here](https://news.ycombinator.com/item?id=18600786) doesn't really shed light on this use case. Maybe @simonbyrne has input?